src/course-outline/outline-sidebar/OutlineSidebar.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Hyperlink } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { useSelector } from 'react-redux';
+
+import HelpSidebar from '../../generic/help-sidebar';
+import { getOutlineDocsLinks } from '../data/selectors';
+import { getFormattedSidebarMessages } from './utils';
+
+const OutlineSideBar = ({ courseId }) => {
+  const intl = useIntl();
+  const docksLinks = useSelector(getOutlineDocsLinks);
+  const sidebarMessages = getFormattedSidebarMessages(docksLinks, intl);
+
+  return (
+    <HelpSidebar
+      intl={intl}
+      courseId={courseId}
+      showOtherSettings={false}
+      className="outline-sidebar mt-4"
+      data-testid="outline-sidebar"
+    >
+      {sidebarMessages.map(({ title, descriptions, link }, index) => (
+        <div className="outline-sidebar-section" key={title}>
+          <h4 className="help-sidebar-about-title">{title}</h4>
```

Comment:
If the parent element has the `outline-sidebar` class, should the child element inherit the general naming principle?
`outline-sidebar-about-title` for example

Reply:
this is the same for other classes on this page
What do you think?

Reply:
This is not my classes, its HelpSidebar classes